atomics/T1055.001/T1055.001.yaml

```
@@ -34,3 +34,51 @@ atomic_tests:
       mavinject $mypid /INJECTRUNNING #{dll_payload}
     name: powershell
     elevation_required: true
+- name: DnsAdmins exploitation (DnsCmd)
+  auto_generated_guid: 3a3e2dfe-9122-4370-8f61-44e1f26c1f7b
+  description: |
+    This module runs the Microsoft tool DnsCmd in order to install a malicious DLL onto a DNS server (this service is usually carried by a Domain Controller).
+    DnsCmd is available either with the Remote Server Administration Tools (https://www.microsoft.com/en-US/download/details.aspx?id=45520) or in "Features on Demand" since Windows 10.
+    Computer must be domain-joined and running with an account that is a member of some specific group (DnsAdmins is expected for the exact exploitation, but this can also work as a Domain Admins).
+    Remark: there is no need for a proper DLL path for triggering the detection. I.e. the registry of the remote server will be updated with this path, whether or not it is valid. But for real exploitation, the DLL has to follow some conditions and the network path needs to be reachable.
```

```suggestion
    Remark: there is no need for a proper DLL path for triggering the detection. I.e. the registry of the remote server (`HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\DNS\Parameters\ServerLevelPluginDll`) will be updated with this path, whether or not it is valid. But for real exploitation, the DLL has to follow some conditions and the network path needs to be reachable.
```

I had this in my notes, I let you double-check. 
Knowing it is helpful for detection engineers who need to write rules :)